### PR TITLE
feat(traveler): enable pagination

### PR DESCRIPTION
**Pull request by yusualhashash**
## How has this been tested?

- [ ] Prod
- [x] Development

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] Added all translations, no hard coded text. 
- [ ] Updated the UI (if needed).
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: apps/web/src/app/[lang]/app/[type]/crm/traveller/table.tsx [hunk 1]
```diff
@@ -27,30 +28,37 @@ export default function Table({
...
+
+  async function fetchTravellerData(page: number) {
+    setLoading(true);
+    try {
+      const response = await getTravellers({
+        maxResultCount: 10,
+        skipCount: page * 10,
+      });
+      if (response.type === "error" || response.type === "api-error") {
+        toast.error(response.status);
```
Comment (a0m0rajab) [inline apps/web/src/app/[lang]/app/[type]/crm/traveller/table.tsx:43]: Let's show a message from the response here as well.

File: apps/web/src/app/[lang]/app/[type]/crm/traveller/table.tsx [hunk 2]
```diff
@@ -27,30 +28,37 @@ export default function Table({
...
+        return;
+      }
+      const Travellersdata = response.data;
+      setTravellers(Travellersdata);
```
Comment (a0m0rajab) [inline apps/web/src/app/[lang]/app/[type]/crm/traveller/table.tsx:47]: No need to have a variable.

**Review by a0m0rajab — CHANGES_REQUESTED** [review]
Minor changes.

File: apps/web/src/app/[lang]/app/[type]/crm/traveller/table.tsx (hunk 2 above)
Comment (yusualhashash) [reply apps/web/src/app/[lang]/app/[type]/crm/traveller/table.tsx:47]: done

File: apps/web/src/app/[lang]/app/[type]/crm/traveller/table.tsx (hunk 1 above)
Comment (yusualhashash) [reply apps/web/src/app/[lang]/app/[type]/crm/traveller/table.tsx]: done

**Review by a0m0rajab — APPROVED** [review]

**Pull request merged**